a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -103,11 +103,19 @@ class AmphoraePostVIPPlug(VThunderBaseTask):
     def execute(self, loadbalancer, vthunder):
         """Execute get_info routine for a vThunder until it responds."""
         try:
-            self.axapi_client.system.action.write_memory()
-            self.axapi_client.system.action.reboot()
-            LOG.debug("Waiting for 30 seconds to trigger vThunder reboot.")
-            time.sleep(30)
-            LOG.debug("Successfully rebooted vThunder: %s", vthunder.id)
+            if "5.2" in vthunder.acos_version[:3]:
```

Comment:
For ACOS version, it may change to 5.3.0, 5.4.0 or 6.0.0...etc. 
Can we find a way to compare version > 5.2 and write an API to do this? 

Here is my thought:
- Provide an API (in util.py or somewhere) to identify the major, minor and patch number in version
For example get_version_number(v1)  that return major, minor, path
- Provide another API which call previous API to compare version string. 
For example acos_version_compare(v1, v2), and returns >0 as v1>v2, =0 as v1==v2 or <0 as v1<v2

Reply:
Done
Added a method `verify_acos_version(acos_version)` to incorporate this logic.